src/test/java/edu/harvard/iq/dataverse/pidproviders/PidUtilTest.java

```diff
@@ -133,6 +145,7 @@ public static void setUpClass() throws Exception {
         pidProviderFactoryMap.put(HandlePidProvider.TYPE, new HandleProviderFactory());
         pidProviderFactoryMap.put(FakeDOIProvider.TYPE, new FakeProviderFactory());
         pidProviderFactoryMap.put(EZIdDOIProvider.TYPE, new EZIdProviderFactory());
+        pidProviderFactoryMap.put(CrossRefDOIProvider.TYPE, new CrossRefDOIProviderFactory());
```

Comment:
Add to the end of the testDOIParsing() test:

```
        String pid7String = "doi:10.11111/DVN/ABCDEF";
        GlobalId pid7 = PidUtil.parseAsGlobalID(pid7String);
        assertEquals(pid7String, pid7.asString());
        assertEquals("crossref1", pid7.getProviderId());
```

That at least tests that the a CrossRef provider gets created by the factory and the authority/shoulder params are being set correctly.

Reply:
added